packages/storybook/cxl-ui/cxl-playbook-accordion/theme=cxl-playbook.faq.stories.js

```diff
@@ -0,0 +1,55 @@
+import { html } from 'lit-html';
```

Comment:
I think can kill off the `faq` part of this story name now, even if it uses cxl-faq.data.json

Reply:
done f1f921c